ReleaseNotes.md

```diff
@@ -4,11 +4,16 @@ This page highlights some changes in the SDK.
 
 Not all changes will be listed, but you can always [compare by version tags](https://github.com/AquaticInformatics/aquarius-sdk-net/compare/v17.2.21...v17.2.25) to see the full source code difference.
 
+### 17.2.29
+
+- Fixed a bug where the SDK would fail to load if the SDK assembly or the application where located in a path containing spaces.
```

Comment:
I think you meant to say "...the SDK assembly or the application is located in a path containing spaces."